caom2utils/caom2utils/caom2blueprint.py

```diff
@@ -4060,29 +4074,29 @@ def augment_temporal(self, chunk):
         # set chunk.time
         self.logger.debug('Begin temporal axis augmentation.')
 
-        aug_naxis = self._get_axis(time_axis_index)
-        aug_error = self._get_coord_error(time_axis_index)
-        aug_ref_coord = self._get_ref_coord(time_axis_index)
-        if self.wcs.has_cd():
-            delta = self.wcs.cd[time_axis_index][time_axis_index]
-        else:
-            delta = self.wcs.cdelt[time_axis_index]
-
         try:
             axis_length = self._get_axis_length(time_axis_index + 1)
         except ValueError:
             self.logger.debug('No WCS Temporal axis.function')
             return
 
-        if aug_ref_coord is not None and axis_length is not None:
-            aug_function = CoordFunction1D(axis_length, delta, aug_ref_coord)
-            naxis = CoordAxis1D(aug_naxis, aug_error, None, None, aug_function)
-            if not chunk.time:
-                chunk.time = TemporalWCS(naxis)
+        if axis_length > 0:
```

Comment:
`axis_length` cannot be `None` anymore?

Reply:
This is to avoid this error https://docs.astropy.org/en/stable/api/astropy.wcs.SingularMatrixError.html when building up the PC/CD matrix from the hdf5 values.

There were no test cases with an `axis_length` of `None`, so I thought it was ok. If there's something different, let me know.